src/post/DetailPost.tsx

```diff
@@ -81,23 +81,22 @@ const DetailPost = () => {
   return (
     <ErrorBoundary>
       <Container>
-        <h1>{title}</h1>
-        <p>작성일: {author.createAt}</p>
+        <Title>{title}</Title>
+        <CreateAt>작성일: {author.createAt}</CreateAt>
         <Div />
         <Author onClick={() => navigate(`/user/${author._id}`)}>
-          <p>작성자: {author.fullName}</p>
           <ProfileImg src={author.image} />
+          <UserName>{author.fullName}</UserName>
         </Author>
         <Div />
+        {image ? <ContentImage src={image} alt='이미지!' /> : null}
```

Comment:
p4;
해당 소스코드로 수정하셔도 될 것 같아요~
```suggestion
        {image && <ContentImage src={image} alt='이미지!' />}
```